tests/transfomers/portable-text-transformer/html-transformer.spec.ts

```diff
@@ -8,190 +8,185 @@ import {
-        const richTextInput = { ...richTextTemplate }
-        richTextInput.value = `<figure data-asset-id="62ba1f17-13e9-43c0-9530-6b44e38097fc" data-image-id="62ba1f17-13e9-43c0-9530-6b44e38097fc"><img src="https://assets-us-01.kc-usercontent.com:443/cec32064-07dd-00ff-2101-5bde13c9e30c/3594632c-d9bb-4197-b7da-2698b0dab409/Riesachsee_Dia_1_1963_%C3%96sterreich_16k_3063.jpg" data-asset-id="62ba1f17-13e9-43c0-9530-6b44e38097fc" data-image-id="62ba1f17-13e9-43c0-9530-6b44e38097fc" alt=""></figure>`;
-        const browserTree = browserParse(richTextInput.value);
-        const nodeTree = nodeParse(richTextInput.value);
-        const nodePortableText = transformToPortableText(nodeTree);
-        const browserPortableText = transformToPortableText(browserTree);
-        const nodeResult = toHTML(nodePortableText, getPortableTextComponents(richTextInput, customResolvers));
-        const browserResult = toHTML(browserPortableText, getPortableTextComponents(richTextInput, customResolvers));
-
-        expect(nodeResult).toMatchSnapshot();
-        expect(nodeResult).toEqual(browserResult);
-    })
-})
\ No newline at end of file
+  const getPortableTextComponents = (
+    element: Elements.RichTextElement,
+    customResolvers: CustomResolvers = {},
+  ): PortableTextOptions => ({
+    components: {
+      types: {
+        image: ({
+          value,
+        }: PortableTextTypeComponentOptions<PortableTextImage>) => {
+          return customResolvers.image
+            ? customResolvers.image(value)
+            : resolveImage(value, toHTMLImageDefault);
+        },
+        component: ({
+          value,
+        }: PortableTextTypeComponentOptions<PortableTextComponent>) => {
+          const linkedItem = element.linkedItems.find(
+            (item) => item.system.codename === value.component._ref,
+          );
+          if (!linkedItem) return `Resolver for unknown type not implemented.`;
+
+          switch (linkedItem.system.type) {
+            case "test":
+              return `<p>resolved value of text_element: <strong>${linkedItem.elements.text_element.value}</strong></p>`;
+            default:
+              return `Resolver for type ${linkedItem.system.type} not implemented.`;
+          }
+        },
+        table: ({
+          value,
+        }: PortableTextTypeComponentOptions<PortableTextTable>) => {
+          return resolveTable(value, toHTML);
+        },
+      },
+      marks: {
+        internalLink: ({
+          children,
+          value,
+        }: PortableTextMarkComponentOptions<PortableTextInternalLink>) => {
+          return `<a href="https://website.com/${value?.reference._ref}">${children}</a>`;
+        },
+        link: ({
+          children,
+          value,
+        }: PortableTextMarkComponentOptions<PortableTextExternalLink>) => {
+          return `<a href=${escapeHTML(value?.href!)}">${children}</a>`;
+        },
+      },
+    },
+  });
+
+  const testConversion = (
```

Comment:
I am not sure if i understand the name of the function.

Reply:
unified the name in all tests